docs/source/getting-started/tracking-server-overview/index.rst

```diff
@@ -0,0 +1,327 @@
+* Enterprise users, or if you want to serve/productionize your model, please use 
+  **Method 3 - production Databricks/AzureML**.
+
+Overall **Method 2 - free hosted tracking server** is the simplest way to get started with MLflow, but please
+pick the method that best suits your needs.
+
+Method 1: Start Your Own MLflow Server
+---------------------------------------
+
+**Disclaimier**: This part of guide is not suitable for running in a cloud-provided IPython environment, e.g., 
+Google Colab or Databricks Notebook. Please follow the guide below in your local machine (laptop/desktop).
+
+Hosted tracking server is the simplest way to store and view MLflow experiments, but it is not suitable for 
+every user. For example, you don't want to expose your data and model to cloud provider. In this case, 
+you can use a local hosted MLflow server to store and view your experiments. To do so, there are two steps:
+
+* Start your MLflow server.
+* Port MLflow session to the local MLflow server IP by :py:func:`mlflow.set_tracking_uri()`.
+
+
+Start a Local MLflow Server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you don't have MLflow installed, please run the command below to install it:
+
+.. code-block:: bash
+
+    $ pip install mlflow
+
+Installing MLflow will also install the MLflow CLI tool, you can start a local MLflow server with UI 
+by running the command below in your terminal:
+
+.. code-block:: bash
+
+    $ mlflow ui
+
+It will generate logs with the IP address, for example:
+
+.. code-block:: bash
+
+    (mlflow) [master][~/Documents/mlflow_team/mlflow]$ mlflow ui
+    [2023-10-25 19:39:12 -0700] [50239] [INFO] Starting gunicorn 20.1.0
+    [2023-10-25 19:39:12 -0700] [50239] [INFO] Listening at: http://127.0.0.1:5000 (50239)
+
+
+Open the URL in your browser will bring you to the MLflow UI. This is the open source version MLflow UI, 
+which is a bit different from the MLflow UI on Databricks CE. Below is a screenshot of the landing page:
+
+.. image:: ../../_static/images/quickstart/tracking-server-overview/mlflow-localhost-landing-page.png
+    :width: 800px
+    :align: center
+    :alt: Landing page of OSS MLflow server
+
+It's also possible to deploy your own MLflow server on cloud platforms, but it is out of the scope of this guide.
+
+
+Connect MLflow Session to Your Server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Now the server is spinned up, let's connect our MLflow session to the local server. This is very 
+similar to how we connect to Databricks platform.
+
+.. code-block:: python
+
+  mlflow.set_tracking_uri("http://localhost:5000")
+
+
+Then let's try logging some dummy metrics, and view them on the local hosted UI:
+
+.. code-block:: python
+
+  mlflow.set_experiment("/check-localhost-connection")
+
+  with mlflow.start_run():
+      mlflow.log_metric("foo", 1)
+      mlflow.log_metric("bar", 2)
+
+
+Putting it together you can copy the following code to your editor and save it as `test_mlflow_localhost.py`:
```

Comment:
```suggestion
Putting it all together, you can copy the following code to your editor and save it as `test_mlflow_localhost.py`:
```

Reply:
can we change the name of the script since files and functions starting with `test` have the connotation of being associated with pytest?

Reply:
renamed to `log_mlflow_with_localhost.py`